### PR TITLE
Add more public point functions for v2 client

### PR DESCRIPTION
**Pull request by sparrc**
cc @jwilder @otoolep

### CODE REVIEW

**Comment by sparrc** [comment]
cc @corylanou

File: client/v2/client_test.go [hunk 1]
```diff
@@ -186,6 +187,54 @@ func TestClient_PointWithoutTimeString(t *testing.T) {
...
+
+	if !reflect.DeepEqual(tags, p.Tags()) {
+		t.Errorf("Error, got %v, expected %v",
+			p.Tags(), tags)
+	}
+}
+
+func TestClient_PointUnixNano(t *testing.T) {
+	const shortForm = "2006-Jan-02"
+	time1, _ := time.Parse(shortForm, "2013-Feb-03")
```
Comment (corylanou) [inline client/v2/client_test.go]: Would it be clearer to do:

```go
unixNanoTime := 1359849600000000000
time1 := time.Unix(0, unixNanoTime)
```

Or some variation of that.

**Comment by corylanou** [comment]
Code looks good, minor comment, but use your judgement.  +1

File: client/v2/client.go [hunk 2]
```diff
@@ -218,6 +218,31 @@ func (p *Point) PrecisionString(precison string) string {
 	return p.pt.PrecisionString(precison)
 }
 
+// Name returns the measurement name of the point
+func (p *Point) Name() string {
+	return p.pt.Name()
+}
+
+// Name returns the tags associated with the point
+func (p *Point) Tags() map[string]string {
```
Comment (otoolep) [inline client/v2/client.go]: Does this return a copy of the tags, or the tags themselves? If the latter, it is mutable by the caller.

Comment (corylanou) [inline client/v2/client.go]: Tags creates a new map every time: https://github.com/influxdb/influxdb/blob/v2client-funcs/models/points.go#L1004

**Comment by otoolep** [comment]
+1

**Pull request closed without merging**